.github/workflows/publish-nuget.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   call-publish-workflow:
```

Comment:
Please fix the job name as it is in the template to "publish-nuget". And everywhere else.

Reply:
done